src/hyp3_autorift/process.py

```diff
@@ -538,6 +584,14 @@ def main():
     g1, g2 = sorted(args.granules, key=get_datetime)
 
     product_file, browse_file = process(g1, g2, parameter_file=args.parameter_file, naming_scheme=args.naming_scheme)
+    thumbnail_file = create_thumbnail(browse_file)
+
+    if args.opendata_upload:
+        bucket = 'its-live-data'
+        prefix = get_opendata_prefix(product_file, args.granules[0])
+        upload_file_to_s3(product_file, bucket, prefix)
+        upload_file_to_s3(browse_file, bucket, prefix)
+        upload_file_to_s3(thumbnail_file, bucket, prefix)
```

Comment:
```suggestion
        prefix = get_opendata_prefix(product_file, g1)
        upload_file_to_s3(product_file, OPEN_DATA_BUCKET, prefix)
        upload_file_to_s3(browse_file, OPEN_DATA_BUCKET, prefix)
        upload_file_to_s3(thumbnail_file, OPEN_DATA_BUCKET, prefix)
```
use `g1` so you're getting the reference scene (or just use the filename; see other comment) and `bucket` should be defined as a constant at the top of the file